less/mixins/grid.less

```diff
@@ -13,8 +13,10 @@
 
 // Creates a wrapper for a series of columns
 .make-row(@gutter: @grid-gutter-width) {
-  margin-left:  ceil((@gutter / -2));
-  margin-right: floor((@gutter / -2));
+  @_margin_left_helpvar:  ceil((@gutter / 2));
```

Comment:
Maybe that's what you meant – I think I understand what you were saying now 😝 
Well, it's updated now – double parenthesis and all, and it works without the "helpvars".